### PR TITLE
[loki] add loki-operator package to CLO's e2e jobs

### PR DESCRIPTION
**Pull request by btaani**
This PR adds the necessary configurations to enable running ci e2e tests on cluster-logging with lokistack

### CODE REVIEW

File: ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml [hunk 1]
```diff
@@ -95,6 +107,13 @@ tests:
 - as: e2e
   steps:
     cluster_profile: gcp
+    dependencies:
+      OO_INDEX: ci-index-loki-operator-bundle
```
Comment (periklis) [inline ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml:111]:
```suggestion
      OO_INDEX: ci-index-cluster-logging-operator-bundle
```

**Review by periklis — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml [hunk 2]
```diff
@@ -65,6 +69,18 @@ images:
 - dockerfile_path: Dockerfile.e2e
   from: elasticsearch-operator-src
   to: cluster-logging-operator-e2e
+operator:
+  bundles:
+  - as: cluster-logging-bundle
+    base_index: operator-index
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
```
Comment (Red-GV) [inline ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml:77]: I think this is missing the `vector` and `logging-view-plugin` substitutions.

File: ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml [hunk 3]
```diff
@@ -65,6 +69,22 @@ images:
...
+  - pullspec: quay.io/openshift-logging/cluster-logging-operator:latest
+    with: pipeline:cluster-logging-operator
+  - pullspec: quay.io/openshift-logging/fluentd:latest
+    with: pipeline:logging-fluentd
+  - pullspec: quay.io/openshift-logging/log-file-metric-exporter:latest
+    with: pipeline:log-file-metric-exporter
+  - pullspec: quay.io/openshift-logging/vector:latest
+    with: pipeline:logging-vector
+  - pullspec: quay.io/openshift-logging/logging-view-plugin:latest
+    with: pipeline:logging-view-plugin
```
Comment (periklis) [inline ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml:87]: This one is missing in the base_images

File: ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml [hunk 4]
```diff
@@ -47,10 +47,18 @@ base_images:
...
+    namespace: logging
+    tag: logging-view-plugin
   oauth-proxy:
     name: "4.11"
     namespace: ocp
     tag: oauth-proxy
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.11
```
Comment (periklis) [inline ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml:61]:
```suggestion
    tag: v4.10
```

**Comment by periklis** [comment]
/pj-rehearse pull-ci-openshift-cluster-logging-operator-master-e2e-target

**Comment by btaani** [comment]
/pj-rehearse pull-ci-openshift-cluster-logging-operator-master-e2e-target

**Comment by hongkailiu** [comment]
/pj-rehearse pull-ci-openshift-cluster-logging-operator-master-e2e-target

**Comment by hongkailiu** [comment]
/pj-rehearse pull-ci-openshift-cluster-logging-operator-master-e2e-target

**Pull request merged**